images/bootstrap/runner.sh

```diff
@@ -123,6 +123,59 @@ if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
   gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}" || true
 fi
 
+#Function to get access token
+get_access_token() {
```

Comment:
Could we create a new script for these functions in project-infra and source it in the prow job? These won't be required by every prow job.